workbench/public/components/QueryResults/QueryResults.test.tsx

```diff
@@ -104,7 +104,7 @@ describe("<QueryResults with data/> spec", () => {
     await fireEvent.click(getAllByRole('tab')[5]);
 
     // It tests sorting
-    await fireEvent.click(getAllByTestId('tableHeaderSortButton')[1]);
+    // await fireEvent.click(getAllByTestId('tableHeaderSortButton')[1]);
```

Comment:
np: here and other places where lines are commented out, can we add `TODO` comments?